README.md

```diff
@@ -14,7 +19,7 @@ Get started with Cloud controller manager for vSphere with Kubeadm with this [qu
 
 ## Quickstart using Helm
 
-Get started with Cloud controller manager for vSphere using Helm with this [Helm quickstart](https://cloud-provider-vsphere.sigs.k8s.io/tutorials/kubernetes-on-vsphere-with-helm.html).
```

Comment:
As a follow-up we should probably fix this link as well. Adding a link to tutorials here should do it https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/SUMMARY.md#tutorials

Reply:
oh wait, I see you fixed it, then I think the old link can be used?

Reply:
yes we could keep using the old link